src/helpers/NetworkManager/postinstall.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+sudo cp src/helpers/NetworkManager/50-org.freedesktop.NetworkManager.rules /etc/polkit-1/rules.d/50-org.freedesktop.NetworkManager.rules
+
+sudo gpasswd -a $USER network
```

Comment:
**[shellcheck]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Double quote to prevent globbing and word splitting. [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086)